src/Microsoft.Data.Analysis/Grouping.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
```

Comment:
You'll need to add the license text up here. Just copying it from another file in the repo will work fine.

Reply:
Never mind, github was showing me an old view